pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1235,9 +1235,7 @@
     dependencies = with self; [ plenary-nvim ];
   };
 
-  neorg = super.neorg.overrideAttrs {
-    dependencies = with self; [ plenary-nvim ];
-  };
+  neorg = neovimUtils.buildNeovimPlugin { luaAttr = "neorg"; };
```

Comment:
```suggestion
  neorg = neovimUtils.buildNeovimPlugin { luaAttr = luaPackages.neorg; };
```

(might need a rebase)

Reply:
I tried and it didn't work. I will try a rebase.

Reply:
It works now !